core/src/test/java/net/consensys/eventeum/integration/broadcast/KafkaBlockchainEventBroadcasterTest.java

```diff
@@ -24,16 +24,16 @@
 import net.consensys.eventeum.integration.KafkaSettings;
 import net.consensys.eventeum.integration.broadcast.blockchain.KafkaBlockchainEventBroadcaster;
 import net.consensys.eventeum.repository.ContractEventFilterRepository;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import org.mockito.ArgumentCaptor;
 import org.springframework.kafka.core.KafkaTemplate;
 
 import java.math.BigInteger;
 import java.util.Optional;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.mockito.Mockito.*;
```

Comment:
For this PR its fine but going forward avoid wild card imports and instead use direct static specific file imports.